addon/components/frost-select-dropdown.js

```diff
@@ -111,6 +110,21 @@ export default Component.extend({
   },
 
   @readOnly
+  @computed('multiselect')
+  /**
+   * The class names for the frost select dropdown secondary options
+   * @param {Boolean} multiSelect - whether or not this is a multiselect
```

Comment:
I think they are both fine, but we should try to be consistent. 

JSDoc doesn't really care :) https://github.com/jsdoc3/jsdoc/issues/1046